todo_app/flask_config.py

```diff
@@ -6,3 +6,4 @@ class Config:
     SECRET_KEY = os.environ.get('SECRET_KEY')
     if not SECRET_KEY:
         raise ValueError("No SECRET_KEY set for Flask application. Did you forget to run setup.sh?")
+    LOGIN_DISABLED = True
```

Comment:
This feels like a poor default for our application! I'd suggest defaulting to False, and potentially allowing an environment variable to override that as appropriate